kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -178,6 +184,15 @@ def get_manifest(self, release_name: str, namespace: str) -> Iterable[HelmTempla
         except ReleaseNotFoundException:
             return ()
 
+    def get_version(self) -> str:
+        command = ["helm", "version", "--short"]
+        short_version = self.__execute(command)
+        version_match = re.search(r"v(\d+\.\d+\.\d+)", short_version)
+        if version_match is None:
+            raise RuntimeError("Could not parse the Helm version.")
+
+        return version_match.group(1)
```

Comment:
consider returning a tuple with major, minor, patch (+ sha or whatever) directly to make it easier to process in the consuming function